examples/listfleetautoscaler.yaml

```diff
@@ -31,15 +31,18 @@ spec:
   fleetName: fleet-example
   policy:
     type: List # List based autoscaling.
-    count:
+    list:
       # The name of the List.
+      # Required.
       key: rooms
-      # Size of a buffer based on the list capacity that is available over the current aggregate list length in the Fleet.
-      # It can be specified either in absolute (i.e. 5) or percentage format (i.e. 5%)
+      # Size of a buffer based on the list capacity that is available over the current aggregate
+      # list length in the Fleet (available capacity).
+      # bufferLength can be specified either in absolute (i.e. 5) or percentage format (i.e. 5%)
+      # Required.
       bufferLength: 5
-      # minimum aggregate list capacity that can be provided by this FleetAutoscaler.
-      # if not specified, the actual minimum capacity will be bufferLength
+      # Minimum aggregate list capacity that can be provided by this FleetAutoscaler.
+      # If minLength is not specified, the actual minimum capacity will be bufferLength.
       minLength: 10
```

Comment:
Do you think if we should change min/maxLength to min/maxCapacity too to keep consistent with the Counter?

Reply:
Makes sense, I'll make that change.